Formula/wso2ei-6.4.0.rb

```diff
@@ -0,0 +1,36 @@
+class Wso2ei640 < Formula
+  desc "WSO2 Enterprise Integrator 6.4.0"
+  homepage "https://wso2.com/integration/"
+  url "https://bintray.com/wso2/binary/download_file?file_path=wso2ei-6.4.0.zip"
+  sha256 "36ab3982cc517674ab1673c1947093bf8022e1f4fb971a2fe6fa1b3085a13917"
+
+  bottle :unneeded
+  depends_on :java => "1.8"
+
+  def install
+    product_name = "wso2ei"
+    product_version = "6.4.0"
+    profiles = %w[integrator business-process broker analytics-dashboard analytics-worker micro-integrator msf4j]
+
+    puts "\nInstalling WSO2 Enterprise Integrator #{product_version}..."
+    profiles.each do |profile|
+      chmod 0755, "bin/#{product_name}-#{product_version}-#{profile}"
+      bin.install "bin/#{product_name}-#{product_version}-#{profile}"
+    end
+
+    libexec.install Dir["*"]
+    bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env("1.8"))
+
+    puts "Installation completed."
+    puts "\nRun the following commands to start each profile of the WSO2 Enterprise Integrator."
+    profiles.each do |profile|
+      puts "\t#{profile}".ljust(20) + ": #{product_name}-#{product_version}-#{profile}"
+    end
+    puts "\nCheers!!"
+    puts
+  end
+
+  test do
+    system "true"
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
With this brew package, we want to just install the WSO2 runtime in the user's filesystem. To make the runtime functional the user has to "start" the server. We do not start the server as part of the installation and give the control of that to the user.

To test the installation we can check for an internal config file within the runtime. Hope that's sufficient to comply with Homebrew standards.

Thanks!